deploy/helm/omni/values.yaml

```
@@ -0,0 +1,68 @@
+
+nameOverride: omni  # optional, if ommitted the Chart name will be used
+debug: false
+domainName: omni.makomedia.nl
+accountUuid: D5C2A21E-7162-4122-B923-A87ED7827F5B
+privateKeySource: "/omni.asc"
+deployment:
+  image: ghcr.io/siderolabs/omni:latest
+  replicaCount: 1
+  annotations:
+    amazing: Annotation
+  imagePullPolicy: IfNotPresent
+auth:
+  auth0:
+    enabled: true
+    clientId: 123456
+    domain: "https://www.auth0.example"
+  saml:
+    enabled: false
+    url: "https://www.saml.example"
+initialUsers:
+ - foo@bar.com
```

This will actually cause the first deploy of Omni to be locked out of the user deploying it. Once an initial use is set, that is the only user that can access Omni :sweat_smile:
```suggestion
initialUsers: []
```